datastore/grantserver/postgres.go

```diff
@@ -87,23 +97,32 @@ func NewPostgres(databaseURL string, performMigration bool, dbStatsPrefix ...str
 	if len(databaseURL) == 0 {
 		databaseURL = os.Getenv("DATABASE_URL")
 	}
+	dbStatsPref := strings.Join(dbStatsPrefix, "_")
+
+	key := dbStatsPref + ":" + databaseURL
+
+	if dbs[key] != nil {
```

Comment:
this part is necessary to prevent too many connections in tests. but useful nonetheless